randovania/data/json_data/prime3.txt

```diff
@@ -3453,6 +4276,13 @@ Asset id: 7063455547734255887
       Trivial
   > Spinner Room
       Spider Ball
+  > Outdoor
+      After Craneyard Windows Broken
+
+> Event - Broken Windows; Heals? False
+  * Event Craneyard Windows Broken
+  > Outdoor
+      Trivial
 
 ----------------
 Creche Transit
```

Comment:
- Add new event called "Creche Transit Hyper Missile Wall"
- "Outside" node can be removed.
- Metroid Processing should connect to:
  - Missile with Boost
  - New event with requirements "Missile, Hyper Missile, HyperModeTank, Suit Type > 4"
  - Metroid Creche with requirement "After new event"
- Metroid Creche should connect to Metroid Processing with requirement "After new event and Suit Type > 4"
- New event should connect to Metroid Creche with requirement "Suit Type > 4"
- Missile should connect to Metroid Processing as Trivial

Reply:
I left the missile tank connect to metroid processing require boost ball since you need boost ball to re-rotate the contraption again anyways. It shouldn't reeeeeally matter but eh.

I also made metroid processing -> metroid creche require both the event and hazard shield

Reply:
I get that. It's just irrelevant to check for Boost when Boost is already needed to get there to begin with.